google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -81,9 +83,17 @@
   private final FlowControlSettings flowControlSettings;
   private final boolean useLegacyFlowControl;
 
+  private AtomicBoolean enableExactlyOnceDelivery;
+
+  private final MessageReceiverWithAckResponse receiverWithAckResponse;
```

Comment:
nit. I made a comment below that we probably don't even need to store this in StreamingSubscriberConnection. But, if you decide you want to still know if a receiverWithAckResponse was provided, I would just make a boolean variable that is set only if a receiverWithAckResponse is provided, rather than storing the receiverWithAckResponse.